recipe/meta.yaml

```diff
@@ -10,14 +10,16 @@ source:
   sha256: a41b1009dd1498ae7d436bfa5ac952a08ca92a4bb9e31dca6e8bb966b49f7fce
 
 build:
-  noarch: python
   number: 0
   script: {{ PYTHON }} -m pip install . -vv
+  skip: True  # [py<36]
 
 requirements:
   host:
     - pip
     - python >=3.6
+    - setuptools
+    - wheel
   run:
     - python >=3.6
     - pyjsparser >=2.5.1
```

Comment:
since `pip` is part of the `requiremennts`, we might want to add `pip` in the test `required` section before we call the `pip check` in the test section.

https://github.com/AnacondaRecipes/js2py-feedstock/blob/8f91676a787d403727a50fc21f3c715dcca8a948/recipe/meta.yaml#L29&L43

Reply:
added `pip` to test `required` section